docs/docs/lib/node.mdx

```diff
@@ -21,20 +21,86 @@ npm install --save @growthbook/growthbook
 
 ## Quick Usage
 
-GrowthBook instances are scoped to a single incoming request. The easiest way to do this is with Middleware:
+First, create and instantiate a singleton GrowthBook client. You can do this as part of your server startup script.
+
+```js
+const client = new GrowthBookMultiUser({
```

Comment:
Would it be better to call `client` something more explicit like `gbClient`? I'm thinking of copying/pasting this code into a large app where `client` might already be defined and where it might be clearer what this client does.